Mathlib/RingTheory/HahnSeries/HEval.lean

```diff
@@ -0,0 +1,117 @@
+/-
+Copyright (c) 2024 Scott Carnahan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Carnahan
+-/
+import Mathlib.RingTheory.HahnSeries.Summable
+import Mathlib.RingTheory.PowerSeries.Basic
+
+/-!
+# A summable family given by a power series
+
+## Main Definitions
+ * `HahnSeries.SummableFamily.powerSeriesFamily`: A summable family of Hahn series whose elements
+   are non-negative powers of a fixed positive-order Hahn series multiplied by the coefficients of a
+   formal power series.
+
+## TODO
+ * `PowerSeries.heval`: An `R`-algebra homomorphism from `PowerSeries R` to `HahnSeries Γ R` taking
+   `X` to a positive order Hahn Series.
+
+-/
+
+open Finset Function
+
+open Pointwise
+
+noncomputable section
+
+variable {Γ Γ' R V α β σ : Type*}
+
+namespace HahnSeries
+
+namespace SummableFamily
+
+section PowerSeriesFamily
+
+variable [LinearOrderedCancelAddCommMonoid Γ] [CommRing R]
+
+variable [CommRing V] [Algebra R V] {x : HahnSeries Γ V} (hx : 0 < x.orderTop) (f : PowerSeries R)
+
+/-- A summable family given by scalar multiples of powers of a positive order Hahn series. -/
```

Comment:
```suggestion
/-- A summable family given by scalar multiples of powers of a positive order Hahn series.

The scalar multiples are given by the coefficients of a power series. -/
```
I think it's surprising enough that it's worth mentioning. I was looking for an `ℕ → R` argument